src/ugeneui/src/plugin_viewer/PluginViewerController.cpp

```diff
@@ -45,16 +45,14 @@ PluginViewerController::PluginViewerController() {
     showServices = false;    //'true' mode is not functional anymore after service<->plugin model refactoring
     mdiWindow = NULL;
     connectStaticActions();
-
-    if (AppContext::getSettings()->getValue(PLUGIN_VIEW_SETTINGS + "isVisible", false).toBool()) {
-        createWindow();
-    }
 }
 
 PluginViewerController::~PluginViewerController() {
     AppContext::getPluginSupport()->disconnect(this);
 
-    AppContext::getSettings()->setValue(PLUGIN_VIEW_SETTINGS + "isVisible", mdiWindow != NULL);
+    // 'true' mode is not functional anymore after service<->plugin model refactoring
```

Comment:
Which service/plugin model refactoring do you mention here?

Reply:
This is an old comment, not my

Reply:
PLUGIN_VIEW_SETTINGS is not used/needed anymore